src/Matrix.cpp

```diff
@@ -283,6 +325,21 @@ FP_PRECISION* Matrix::getA() {
 
 
 /**
+ * @brief Get the A component (excluding the diagonal) of the CSR form of the
```

Comment:
I added more description on the A component and changed the AD component to LU (since it is really the lower + upper components of the matrix). I also added a reference to the wikipedia page where the A, IA, and JA components are described.